errors/code.go

```diff
@@ -0,0 +1,71 @@
+package errors
+
+import (
+	"context"
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func Convert(err error) *status.Status {
+	if err == nil {
+		return status.New(codes.OK, "OK")
+	}
+
+	if se, ok := err.(interface {
+		GRPCStatus() *status.Status
+	}); ok {
+		return se.GRPCStatus()
+	}
+
+	switch err {
+	case context.DeadlineExceeded:
+		return status.New(codes.DeadlineExceeded, err.Error())
+	case context.Canceled:
+		return status.New(codes.Canceled, err.Error())
+	}
+
+	return status.New(codes.Unknown, err.Error())
+}
+
+func GrpcToHTTPStatusCode(statusCode codes.Code) int {
```

Comment:
I noticed this. Perhaps different from our specifications.
```diff
- func GrpcToHTTPStatusCode(statusCode codes.Code) int {
+ func GRPCToHTTPStatusCode(statusCode codes.Code) int {
```

Reply:
fix done!